gcp/microservice-ecommerce/xebialabs/xlr-pipeline-ci-cd.yaml.tmpl

```diff
@@ -0,0 +1,216 @@
+{{$app := .AppName | kebabcase}}
+{{$clusterName := .ClusterName | kebabcase}}
+{{/* TODO change to new environment once XLD issue is fixed */}}
+{{/* $appEnv := printf "gcp-gke-%s" $app */}}
+{{$appEnv := printf "gcp-terraform" }}
+apiVersion: xl-release/v1
+kind: Templates
+spec:
+- name: {{$app}}
+  type: xlrelease.Folder
+  children:
+  - name: {{$app}}-ci-cd
+    type: xlrelease.Release
+    description: |
+      This XL Release template shows how to deploy an application, based on microservices architecture, to GCP GKE using XL Deploy and Terraform.
+    tags:
+    - GCP
+    - GKE
+    - Terraform
+    - {{$app}}
+    scriptUsername: !value XL_RELEASE_USERNAME
+    scriptUserPassword: !value XL_RELEASE_PASSWORD
+    variables:
+    - key: control
+      type: xlrelease.MapStringStringVariable
+      requiresValue: false
+      showOnReleaseStart: false
+      value:
+        namespace: {{.Namespace}}
+        serviceName: store
+    - key: lbHostnameOrIp
+      type: xlrelease.StringVariable
+      requiresValue: false
+      showOnReleaseStart: false
+      
+    - key: store-admin-username
+      type: xlrelease.StringVariable
+      requiresValue: true
+      showOnReleaseStart: false
+      value: {{.StoreAdminUsername}}
+    - key: store-admin-password
+      type: xlrelease.StringVariable
+      requiresValue: true
+      showOnReleaseStart: false
+      value: {{.StoreAdminPassword}}
+    - key: xld-admin-password
+      type: xlrelease.PasswordStringVariable
+      requiresValue: true
+      showOnReleaseStart: false
+      value: !value XL_DEPLOY_PASSWORD
+    phases:
+    {{if eq .ProvisionCluster "true"}}
+    # Provision Infra
+    - name: Provision Infrastructure
+      color: '#ff9e3b'
+      type: xlrelease.Phase
+      tasks:
+      - name: Provision GCP GKE cluster
+        type: xldeploy.Deploy
+        server: XL Deploy
+        deploymentPackage: {{$app}}/GKE-TERRAFORM/{{$app}}-terraform-gke/1.0.0
+        deploymentEnvironment: Environments/{{$app}}/gcp-terraform
+    {{end}}
+    - name: Build {{$app}} application
+      type: xlrelease.Phase
+      tasks:
+      - name: Build docker images for {{$app}} services
+        type: xlrelease.ParallelGroup
+        tasks:
+        - name: Build {{$app}} store docker image
+          type: jenkins.Build
+          jenkinsServer: jenkins
+          jobName: devops-as-code-eks-blueprint-store-svc
```

Comment:
eks -> gke

Reply:
it wouldn't work since the Jenkins jobs are named like that, we need to change the name in the e-commerce repo and sync all blueprints with it, then it might also break current blueprints and docs, so I decided to keep it as it is for now as its trivial